i18n/en-US/articles/context-environment.md

```diff
@@ -44,6 +45,29 @@ export default Page;
 
 The environment *key* is an md5 hash of the current environment folder outputs. The key is appended to [assets](/styles) and [static API](/static-site-generation) path to assist cache control.
 
+The environment *hot* is boolean that identifies if hot reload is enabled and is available only in development mode.
+
+## Custom Events
+
+During development any updates to tracked files will raise a custom event you can use to facilitate development flow.
+
```

Comment:
Maybe we could explain what is the direct consequence of it?
Like "with this, you can [...]" and then explain what's the point of the code below? I am not sure if it is 100% clear.